Tests/StarknetTests/Data/TypedDataTests.swift

```diff
@@ -421,7 +421,7 @@ final class TypedDataTests: XCTestCase {
             (
                 Self.CasesRev0.tdValidate,
                 "0xcd2a3d9f938e13cd947ec05abc7fe734df8dd826",
-                "0x6038f35de58f40a6afa9d359859b2f930e5eb987580ba6875324cc4dbfcee"
+                "0x3b3fbdb1961ae09cb5b2716b07142e63930fb2eddc4842641ca1df7352766a1"
```

Comment:
Expected value has been changed here because of the change in [typed_data_rev_0_validate_example.json](https://github.com/software-mansion/starknet.swift/pull/192/files#diff-13ac7bca51393bcf87abc99260ec12e87bf315b876c15c5cd11ba5c076a8b61aR67)